src/components/SingleDatePicker.jsx

```diff
@@ -211,20 +212,22 @@ class SingleDatePicker extends React.Component {
     });
 
     onFocusChange({ focused: false });
-    onClose({ startDate, endDate });
+    onClose({ date });
```

Comment:
hm, since `onClose` is user code, this is a breaking change as well.

Reply:
Bugger.  Ok I'll move it over!